### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by emanueledimauro** [cifkao/html-midi-player#18] [Q&A]
Very useful code !!!!
I have a question about API utilisation. 
I use html code in order to make my player : 

<midi-player
  src="https://magenta.github.io/magenta-js/music/demos/melody.mid"
  sound-font visualizer="#myVisualizer">
</midi-player>
<midi-visualizer type="piano-roll" id="myVisualizer"></midi-visualizer>

But I don't know How use API to get information (in my case I need currentTime() function to try to scroll svg piano roll visualizer). 
Thanks for your good work !
Emanuele

**Comment by cifkao** [comment]
I don't see any code in your post. Anyway, `currentTime` is a property, so you just need `myPlayer.currentTime` (where `myPlayer` is your player element). You can also listen to the `'note'` event using `myPlayer.addEventListener('note', listener)` to get notified about the currently playing note.

Let me know if this answers your question!
